README.md

```diff
@@ -19,7 +19,7 @@ To deploy the war do:
 
 By default, log messages are redirected to the `STDOUT`.
 
-The api documentation is located at `<your-server-address>:<port>/radar/api/openapi.json`
+The api documentation is located at `<your-server-address>:<port>/api/openapi.json`
```

Comment:
I think we should also change the url in restapi-details.yml in integrationTest to match this.

Reply:
Very nice and neat work. I will start merging this branch with the `removing-device-catalog` branch which also has significant amount of refactoring and will result in large merge conflict. Once approved we will update from dev.